wordpress-mobile/WordPress-iOS
# Make sure changes to media are only done when they are real changes.

This PR adds checks for the media update method to only trigger changes on CoreData if a change was made server side.

To test:
 - Open the app
 - Go to a media library of a site.
 - Move around to see if there is no strange behaviour
 - Add new media to the library.
